libs/utils/WordPressUtils/src/androidTest/java/org/wordpress/android/util/PhotonUtilsTest.java

```diff
@@ -0,0 +1,104 @@
+package org.wordpress.android.util;
+
+import org.junit.Test;
+import org.wordpress.android.util.PhotonUtils.Quality;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class PhotonUtilsTest {
+    @Test
+    public void getPhotonImageUrlIsEmptyWhenUrlIsNull() {
+        String photonUrl = PhotonUtils.getPhotonImageUrl(null, 0, 1);
+
+        assertEquals("", photonUrl);
```

Comment:
I'd rather use `assertThat(photonUrl).isEmpty()` and generally use `assertThat` than other asserts. I think it's more readable and I like the consistency.

Reply:
Agree. AFAIU in the WPUtils we use JUnit and Hamcrest, so I moved using that assertThat pattern here 9881777ae35ad408285b5452a5639248e79b543d 😊